share/spice/currency/currency.css

```diff
@@ -205,6 +205,12 @@
     }
 }
 
+@media only screen and (max-width: 600px) {
```

Comment:
Thank you for that feedback @zaahir. I'll go ahead and work in those
changes.

On Tue, Mar 29, 2016 at 3:48 PM, Zaahir Moolla <notifications@github.com>
wrote:

> In share/spice/currency/currency.css
> <https://github.com/duckduckgo/zeroclickinfo-spice/pull/2628#discussion_r57797723>
> :
>
> > @@ -205,6 +205,12 @@
> >      }
> >  }
> >
> > +@media only screen and (max-width: 600px) {
>
> Looks like this is a little too restrictive. LandscapeiPhone 6Plus is 700+
> px wide:
>
> [image: 299_usd_at_duckduckgo]
> <https://cloud.githubusercontent.com/assets/873785/14123280/cb020028-f5cd-11e5-8b90-96c0df74f286.png>
>
> It may be better to use our is_mobile js variable to set a mobile class
> that sets a width on the image, regardless of the device width.
>
> Alternatively you can try targeting .is-mobile .flag_img -- we add an
> is-mobile class to <html> on mobile devices [image: :+1:]
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/duckduckgo/zeroclickinfo-spice/pull/2628/files/aabab77588b52f5636a6e142a7dd6669f771d5e4#r57797723>
>